tests/console/snapper_create.pm

```diff
@@ -19,7 +19,18 @@ sub get_last_snap_number {
     # In many cases script output returns not only script execution results
     # but other data which was written to serial device. We have to ensure
     # that we got what we expect. See poo#25716
-    my $output = script_output("snapper list | tail -n1 | awk '{ print \">>>\" \$3 \"<<<\" }'");
+
+    # get snapshot id column
+    my $snap_head         = script_output("snapper list | head -n1");
+    my $snap_id_col_index = 1;
+    for my $field (split(/\|/, $snap_head)) {
+        $field =~ s/^\s+|\s+$//g;    # trim spaces
+        last if ($field eq '#');     # get snapshot id field
+        $snap_id_col_index++;
+        die "Unable to determine snapshot id column index" if ($snap_id_col_index > 50);
```

Comment:
This won't work, as we don't have that many columns. Ultimate check is the match or at least that we got index which is bigger than number of columns.

Reply:
resolved that.